### PR TITLE
Post initial implementation

### PR DESCRIPTION
**Pull request by Joergenson**
All visuals should be ok now. It only needs post functionality. 😎

### CODE REVIEW

**Review by kepet19 — APPROVED** [review]

File: components/breeds.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      {
+        name: 'Husky',
+        id: 4,
+      },
+      {
+        name: 'Something that says woof idk',
+        id: 5,
+      },
+      {
+        name: 'YEP',
```
Comment (Kechr) [inline components/breeds.js:35]:
```suggestion
        name: 'Corgi',
```

**Review by Kechr — COMMENTED** [review]
Looking sweet, nice job!

File: components/breeds.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      {
+        name: 'Golde reciever',
+        id: 1,
+      },
+      {
+        name: 'Cancer',
+        id: 2,
+      },
+      {
+        name: 'Rat',
```
[the remark this answers is not in the document]
Comment (Joergenson) [reply components/breeds.js:23]: u sure about that one :)?

File: components/breeds.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        id: 3,
+      },
+      {
+        name: 'Husky',
+        id: 4,
+      },
+      {
+        name: 'Something that says woof idk',
```
[the remark this answers is not in the document]
Comment (Joergenson) [reply components/breeds.js:31]: close enough

File: components/breeds.js (hunk 1 above)
Comment (Joergenson) [reply components/breeds.js:35]: Same same

**Comment by kepet19** [comment]
har i lige lavet 6 commits for at ændre navne? xD

**Review by Kechr — APPROVED** [review]

**Comment by Kechr** [comment]
> har i lige lavet 6 commits for at ændre navne? xD

Kun 5, lol

**Comment by kepet19** [comment]
> > har i lige lavet 6 commits for at ændre navne? xD
> 
> Kun 5, lol

haha det er sjovt aligevel xD Ahh simon cleared it up!

**Comment by Joergenson** [comment]
Should be good now

**Pull request merged**